### DISCUSSION TITLE
How to create Keygen for Tekla SD Software

### DISCUSSION THREAD

**Discussion by zakmuh** [keygen-sh/community#107] [General]
Hi There,

I'd like to learn how creating keygen application which would generate licence file for Tekla sturctural designer software. 

The TSD software I'm using is downloaded and it comes with a keygen for crack. I have to run it and create the Local.LICDB file and then paste it into the Licence folder in Programe Date file in Windows C drive. This file only works for 12 months. Changing the date through Notepad editor not extending the licence. 

Is it possible to make this Keygen software by myself so that I can create a new for new version of TSD?

Please see below a screenshot of LICDB file opened in Notepad....

![image](https://github.com/keygen-sh/community/assets/147681437/eba7b96d-a882-4199-91f7-44154f044c1f)

![image](https://github.com/keygen-sh/community/assets/147681437/0eb80f03-f6a4-4e52-9d3e-ed3851cb3445)

![image](https://github.com/keygen-sh/community/assets/147681437/b1dd5398-ffb2-4d71-8885-b6076b7ca35b)

I'd highly appreciate if anyone could advise me and put me on the right direction.

Cheers

Zak

**Comment by ezekg** [comment]
Keygen does not help with illegally cracking software.
